client/Elements/Components/TopicConsumer/SearchTopicsConsumerList.patternfly.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
```

Comment:
Can we rename 'Consumer' to 'ConsumerGroups' everywhere, as consumer could be a bit misleading?